### DISCUSSION TITLE
Port pinouts ?

### DISCUSSION THREAD

**Discussion by niallp** [CharlesGodwin/pymagnum#52] [Q&A] [answered]
Reviewing the docs it looks like RS-485 is on the second pair of the RJ11 and I was wondering if anyone has figured out the functions on the other pairs ? I've seen the availability of the remote start pigtail (ME-RSA and ME-RSA-M) which connects on the stacking port but trying to figure out if I can use this from a controller (without an RS-485 interface) to turn on/off the inverter function.

**Comment by CharlesGodwin** [comment, marked as answer]
I'll start by saying I have no knowledge of the wiring on the Stack (Red) interface on the inverters. I have read the manual from Magnum and I'm no wiser. Based on their users manual, as long as anything you connect to the pigtail operates as expected, it should work.

The answer to your question about what is on the other pair on the Magnum Net (Green) interface. 
The documentation says that it is this:

From the top on the inverter:
1 = "B" RS485 comm
2 = +14VDC
3= Ground
4 = "A" RS485 comm

It also say not to use the 14VDC as there is low current. I think it powers the Remotes etc. but not the AGS-N unit with 3 relays. The AGS-N uses separate DC power.

I see the that you can buy ME-RSA and ME-RSA-M for $40 Canadian. I suggest you buy one and connect whatever you want, including computer controlled switching, to the end.

**Reply by niallp** [reply]
Thanks, with further research and some testing I found the magic combo to emulate the ME-RSA-M is 1+6 (outer pair) connected through momentary switch to pin 3 (which would correspond to +14 Vdc if the stacking port duplicated the Magnum Net port but I didn't see any voltage on it wrt pin 4)

Might help anyone looking to avoid spending $40 for a 50 cent RJ12 ... ;)
